src/test/unit/controls/Graph/Graph-spec.js

```diff
@@ -2926,4 +2936,27 @@ describe("Graph", () => {
             expect(toNumber(translate[1], 10)).toBeCloserTo(10);
         });
     });
+    describe("When the value of the data is not present", () => {
+        it("displays No Data, if showNoDataText is set to true or undefined", () => {
+            const primaryContent = new Line(getData());
+            graph = new Graph(getAxes(axisDefault));
+            graph.loadContent(primaryContent);
+            const noDataTextElemtn = fetchElementByClass(styles.noDataText);
+            expect(noDataTextElemtn).not.toBeNull();
+        });
+        it("doesn't displays No Data, if showNoDataText is set to false", () => {
+            const primaryContent = new Line(getData());
+            graph = new Graph(
+                Object.assign(
+                    {
+                        showNoDataText: false
+                    },
+                    getAxes(axisDefault)
+                )
+            );
+            graph.loadContent(primaryContent);
+            const noDataTextElemtn = fetchElementByClass(styles.noDataText);
+            expect(noDataTextElemtn).toBeNull();
```

Comment:
I would add unit tests for 
* Resize
* Load
* Unload 
* When the `height` property is used to change the graph height, the no data text should still be centered vertically.

Reply:
Added necessary test cases.